src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java

```diff
@@ -516,6 +516,9 @@ public enum State {
         Rejected
     ));
 
+    public static final EnumSet<State> POST_ISSUE_STATES_WITH_ARCHIVED =
+            EnumSet.complementOf(EnumSet.copyOf(POST_ISSUE_STATES).of(Archived));
```

Comment:
Did you try removing ARCHIVED from POST_ISSUE_STATES? I think we might be able to do that and avoid creating this variable?

If we leave the variable, should it be `POST_ISSUE_STATES_WITHOUT_ARCHIVED`?